iOS/UI/Reader/ReaderToolbarView.swift

```diff
@@ -8,7 +8,19 @@
 import UIKit
 
 class ReaderToolbarView: UIView {
-
+    var currentPageValue: Int? {
+        didSet {
+            if previousPageValue != currentPageValue {
+                previousPageValue = currentPageValue
+            }
+        }
+    }
+    var previousPageValue: Int? {
+        willSet {
+            let feedbackGenerator = UISelectionFeedbackGenerator()
+            feedbackGenerator.selectionChanged()
+        }
```

Comment:
you don't need two variables, just use `oldValue`
```suggestion
            if oldValue != currentPageValue {
                let feedbackGenerator = UISelectionFeedbackGenerator()
                feedbackGenerator.selectionChanged()
            }
        }
```